domain-ee/ee-ep-merge-app/src/python_src/api.py

```diff
@@ -52,5 +62,56 @@ def get_health_status():
     return {"status": "ok"}
 
 
+@app.post("/merge",
+          status_code=status.HTTP_202_ACCEPTED,
+          response_model=MergeEndProductsResponse,
+          response_model_exclude_none=True)
+async def merge_claims(merge_request: MergeEndProductsRequest, background_tasks: BackgroundTasks):
+    if validate_merge_request(merge_request):
+        job_id = uuid4()
+        logging.info(f"event=mergeJobSubmitted "
+                     f"job_id={job_id} "
+                     f"pending_claim_id={sanitize(merge_request.pending_claim_id)} "
+                     f"supp_claim_id={sanitize(merge_request.supp_claim_id)}")
+
+        merge_job = MergeJob(job_id=job_id,
+                             pending_claim_id=merge_request.pending_claim_id,
+                             supp_claim_id=merge_request.supp_claim_id)
+        job_store.submit_merge_job(merge_job)
+
+        return jsonable_encoder({"job": merge_job})
+    else:
+        raise HTTPException(status_code=400, detail="Claim IDs must be different.")
```

Comment:
If you convert this raise into an early short circuit, you can save on a bunch of indentation in the success pathway 😉